test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -445,12 +445,12 @@ void UcxContext::progress_io_message()
     } else {
         UcxConnection *conn = iter->second;
         if (!conn->is_established()) {
-            // tag-recv request can be completed before stream-recv callback has
-            // been invoked, go to another progress round and handle this io-msg
-            return;
-        }
-
-        if (conn->ucx_status() == UCS_OK) {
+            // tag-recv request can be completed before stream-recv callback
+            // has been invoked, cache this operation to handle it when
```

Comment:
cache -> defer the processing of io message to the point when connection is established

Reply:
done